src/components/Modals/RulesModal.vue

```diff
@@ -116,25 +116,130 @@
                     </li>
                     <li><h5>Hack</h5>
                       <div>
-                        <img src="../../static/cardImg/H.png" alt="Instruction 1 Card" class="cardImage">
+                        <img src="../../../static/cardImg/H.png" alt="Instruction 1 Card" class="cardImage">
                       </div>
 
                       <div>
                         <ol>
                           <li>Hack cards let you attack another player's program.</li>
                           <li>When you play a Hack card, you choose which stack of cards from another player's playing
                             fields will be destroyed. </li>
-                          <li>You cannot hack a Group card.</li>
+                          <li>You cannot hack a Group card or someone with a Firewall card active.</li>
                         </ol>
                         <b>Relation to computer programming:</b>
-                        Sometimes a mean or mischievous computer programmer. known as a <i>hacker</i> will change a computer program
+                        Sometimes a mean or mischievous computer programmer. Known as a <i>hacker</i> will change a computer program
                         to do something it is not supposed to do. Using functions can make it harder for a hacker to change the computer program,
                         which is why Group cards counter Hack cards in the game.
-                      </a>
+
+                      </div>
+                    </li>
+                    <li><h5>Virus</h5>
+                      <div>
+                        <img src="../../../static/cardImg/Virus.png" alt="Instruction 1 Card" class="cardImage">
+                      </div>
+
+                      <div>
+                        <ol>
+                          <li>Virus cards let you infect another player's program, cutting their score in half.</li>
+                          <li>When you play a Virus card, you choose which player will be infected. </li>
+                          <li>You cannot infect someone with a AntiVirus card active.</li>
```

Comment:
"_an_ Antivirus card."

Do we need to say "active"? Isn't just that once the player has played these "safety" cards, that they remain in the effect for the rest of the game?

Reply:
I worded this incorrectly, I should have said an AntiVirus played.